src/olympia/zadmin/views.py

```diff
@@ -301,6 +302,8 @@ def validation_summary(request, job_id):
 def validation_summary_affected_addons(request, job_id, message_id):
     addons = ValidationResultAffectedAddon.objects.filter(
         validation_result_message=message_id)
+    # Get rid of duplicates
+    addons = list(set(addons))
```

Comment:
The name `addons` is probably misleading here sine it contains `ValidationResultAffectedAddon` instances and not actual add-ons. It's probably easier to filter it in a for-loop to be sure.